src/functionalTest/java/uk/gov/hmcts/probate/functional/tests/FormsFunctionalTests.java

```diff
@@ -2,146 +2,256 @@
+                .when()
+                .post(FORMS_NEW_CASE)
+                .then()
+                .assertThat()
+                .statusCode(200)
+                .body("applications.ccdCase.id", hasItem(notNullValue()))
+                .body("applications.ccdCase.state", hasItem(CCD_CASE_STATE_PENDING))
+                .body("applications.caseType", hasItem(ProbateType.INTESTACY.toString()))
+                .body("applications.dateCreated", hasItem(currentDate));
+    }
 
-        IdamData idamData = IdamData.builder()
-            .email(email)
-            .forename(forename)
-            .surname(surname)
-            .password(PASSWORD)
-            .userGroupName(USER_GROUP_NAME)
-            .build();
+    @Test
+    @Pending
+    public void initiateFormsNewCaseForProbateTypeCAVEAT() {
+        RestAssured.given()
+                .relaxedHTTPSValidation()
+                .headers(utils.getCitizenHeaders())
+                .queryParam("probateType", ProbateType.CAVEAT)
+                .when()
+                .post(FORMS_NEW_CASE)
+                .then()
+                .assertThat()
+                .statusCode(200)
+                .body("applications.ccdCase.id", hasItem(notNullValue()))
+                .body("applications.ccdCase.state", hasItem(CCD_CASE_STATE_PENDING))
+                .body("applications.caseType", hasItem(ProbateType.CAVEAT.toString()))
+                .body("applications.dateCreated", hasItem(currentDate));
+    }
 
+    @Test
+    public void InitiateFormsNewCaseWithInvalidQueryParam() {
         RestAssured.given()
-            .relaxedHTTPSValidation()
-            .headers(Headers.headers(new Header("Content-Type", ContentType.JSON.toString())))
-            .baseUri(idamUrl)
-            .body(objectMapper.writeValueAsString(idamData))
-            .when()
-            .post("/testing-support/accounts")
-            .then()
-            .statusCode(201);
+                .relaxedHTTPSValidation()
+                .headers(utils.getCitizenHeaders())
+                .queryParam("probateType", INVALID_PROBATE_TYPE)
+                .when()
+                .post(FORMS_NEW_CASE)
+                .then()
+                .assertThat()
+                .statusCode(400);
+    }
 
+    @Test
+    public void getAllFormsCases() {
+        RestAssured.given()
+                .relaxedHTTPSValidation()
+                .headers(utils.getCitizenHeaders())
+                .when()
+                .get(FORMS_GET_ALL_CASES)
+                .then()
+                .assertThat()
+                .statusCode(200);
     }
 
     @Test
-    @Pending
-    public void shouldCreateDraftThenSubmitAndFinallyUpdatePayment() throws IOException {
-        shouldSaveFormSuccessfully();
-        shouldUpdateForm();
-        shouldSubmitForm();
-        shouldUpdatePaymentSuccessfully();
+    public void shouldCreateDraftThenSubmitAndFinallyUpdatePayment() throws IOException, JSONException {
```

Comment:
Should this test be here if it's commented out?

Reply:
As this is existing broken test which I try to fixed it but looks like flaky test, need some more time to investigate and fix it. I will add @Pending tag to the test.